### PR TITLE
Update FRC-Char generation docs

### PR DESCRIPTION
**Pull request by Piphi5**
Adds documentation for the Control Type selector that was added to the FRC-Characterization tool.

Also updates generation and deployment images to show the new Project Creator GUI.

### CODE REVIEW

File: source/docs/software/wpilib-tools/robot-characterization/generating-project.rst [hunk 1]
```diff
@@ -11,12 +11,19 @@ First, select the desired project location on the new project GUI:
...
-.. note:: The project type dropdown now chooses between the different types of characterization tests as the previous motor setups have been integrated into the config file.
+.. note:: The project type dropdown now chooses between the different types of characterization tests as the previous motor setups have been integrated into the ``Control Type`` field.
 
 Configure Project Parameters
 ----------------------------
 
-In order to run on your robot, the tool must know some parameters about how your robot is set up. Project config settings are formatted as a `Python dictionary literal <https://docs.python.org/3/library/stdtypes.html#mapping-types-dict>`__. These can be modified via the in-window config editor:
+In order to run on your robot, the tool must know some parameters about how your robot is set up.
+
+First, you need to use the ``Control Type`` field to select the appropriate project config template. ``Simple`` is for PWM Based motor controllers, ``CTRE`` is for CAN connected CTRE Motor Controllers (e.g. Talon SRX), and ``SparkMax`` is for the Spark Max Motor Controller. This allows you to fill out the parameters specific to the type of controllers you are using.
```
Comment (Daltz333) [inline source/docs/software/wpilib-tools/robot-characterization/generating-project.rst:21]: Should we use ``:guilabel:`` here?

File: source/docs/software/wpilib-tools/robot-characterization/generating-project.rst [hunk 2]
```diff
@@ -26,6 +33,7 @@ Take care of the following caveats when entering your robot specifications:
 - The key names must *not be changed*, as they are hard-coded for each project type. Only the values (i.e. the things on the right-hand side of the colons) should be modified.
 - ``True`` and ``False`` *must* be capitalized, as they are evaluated as native Python.
 - All string values (e.g. controller names and unit types) *must* be wrapped in quotes and *must* correspond exactly to one of the options described.
+- Read the comments provided in the config file carefully.
```
Comment (Daltz333) [inline source/docs/software/wpilib-tools/robot-characterization/generating-project.rst:36]: This should be an ``.. important::`` directive

File: source/docs/software/wpilib-tools/robot-characterization/generating-project.rst (hunk 1 above)
Comment (Piphi5) [reply source/docs/software/wpilib-tools/robot-characterization/generating-project.rst:21]: Yeah we probably should

**Comment by Piphi5** [comment]
On closer inspection, there are a lot of places that need GUI Labels so I'll add that onto this PR.

**Review by Daltz333 — APPROVED** [review]

**Comment by Daltz333** [comment]
@prateekma does all the images look right to you?

**Review by prateekma — APPROVED** [review]

**Pull request merged**